docs/source/quickstart.rst

```diff
@@ -26,6 +26,23 @@ At the end of these short instructions, you will be able to run the following sn
         sleep(250) # wait 250 msec
         pixels_off() # turn off the GiggleBot eyes
         sleep(250) # wait 250 msec
+
+
+.. note::
+
+    There are 2 MCUs sitting on the BBC micro:bit:
+
+    * One MCU has the role of flashing the HEX file that gets copied over to the other MCU.
+    * And the second MCU houses the actual scripts and MicroPython firmware that you interact with (in the real world or at the terminal).
+
+    Both of these MCUs have their own firmware, but sometimes the firmware sitting on the MCU that hosts the scripts and the MicroPython firmware gets to be called a *runtime*. Other
```

Comment:
In the micropython ecosystem, people refer to it as _firmware_ .  In the micro:bit ecosystem, it is referred to as _micropython runtime_  to differentiate it from the microbit firmware, also known as the DAL firmware.